pkg/controller/jenkins/reconciler_test.go

```diff
@@ -0,0 +1,181 @@
+package jenkins
+
+import (
+	"testing"
+	"time"
+
+	"reflect"
+
+	"github.com/redhat-developer/openshift-jenkins-operator/test/mocks"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	resource "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	test_ns   = "test"
+	test_name = "test-jenkins"
+)
+
+// func TestReconcilerCreateResourceIfNotPresent(t *testing.T) {
```

Comment:
Small point - and I am frequently guilty of this too  ;-)  - we should never have commented out code in any programs, tests, etc.